src/arc_utilities/repeater.py

```diff
@@ -0,0 +1,27 @@
+import asyncio
+from threading import Timer
+from typing import Type
+
+import rospy
+
+
+class Repeater:
```

Comment:
This class does not actually seem to be a repeater.
1) It never starts
2) It just creates a single timer, which runs once.

This seems more like a delayed publisher